src/pages/FormBuilder/FormBuilder.module.css

```
@@ -26,11 +26,9 @@
   display: flex;
   flex-direction: column;
   background: var(--gray-100);
+  width: 100%;
   font-family:
-    "Segoe UI",
-    system-ui,
-    -apple-system,
-    sans-serif;
+    "Merriweather", serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Merriweather" (font-family-name-quotes)

(font-family-name-quotes)